NAMESPACE

```diff
@@ -101,6 +101,7 @@ export(tidy_lymphgen)
 export(web_initialize_gambl_site)
 import(ComplexHeatmap)
 import(DBI)
+import(GenomicRanges)
```

Comment:
Can you please give an idea on what the GenomicRanges are used for? If it is just to overlap the coordinates, we can consider using the foverlaps and avoid adding another big-sized dependecy

Reply:
Indeed, the `GenomicRanges` package is used for intersecting two bed-like files. I will revert this to using `foverlaps` as suggested above. Thanks!